### PR TITLE
LPS-77241 - Compat modal footer buttons separation

### PR DESCRIPTION
**Pull request by jbalsas**

### CODE REVIEW

**Comment by jbalsas** [comment]
ci:test

**Comment by jbalsas** [comment]
Preliminary tests ran on https://github.com/jbalsas/liferay-portal/pull/979#issuecomment-360687209

It should be fine, but I'm running them in here just in case

**Comment by jbalsas** [comment]
Hey @jpince, this is running well locally, but failed here... I'm retesting, could you guys take a look if it fails again?

**Comment by jbalsas** [comment]
ci:retest

**Comment by jpince** [comment]
@Ithildir any ideas why this might have failed?

**Comment by jpince** [comment]
ci:close

**Comment by Ithildir** [comment]
@jpince, this is a Gulp error so I'm not sure how much I'll be able to help... I'll have a look anyway.

File: modules/apps/foundation/frontend-js/frontend-js-web/build.gradle [hunk 1]
```diff
@@ -30,12 +30,6 @@ buildClay {
 
 	eachFile new StripPathSegmentsAction(3)
```
Comment (Ithildir) [inline modules/apps/foundation/frontend-js/frontend-js-web/build.gradle]: @jbalsas, can you remove this empty line?

**Comment by Ithildir** [comment]
@jpince, @jbalsas, did you forget to update the `frontend-theme-unstyled/package-lock.json` file? I don't see it in the commits, and maybe that's why it's failing on CI: its hash didn't change, and so CI simply copied the old `node_modules` dir which still contains the previous version of Clay.

File: modules/apps/foundation/frontend-js/frontend-js-web/build.gradle (hunk 1 above)
Comment (jbalsas) [reply modules/apps/foundation/frontend-js/frontend-js-web/build.gradle]: Of course!

**Comment by jbalsas** [comment]
You're totally right, @Ithildir, nice catch!! 👏 

Thanks!

**Pull request closed without merging**